libs/features/mas/src/merch-card-collection.js

```diff
@@ -93,27 +93,30 @@ const searcher = (elements, { search }) => {
 
 export class MerchCardCollection extends LitElement {
     static properties = {
+        displayResult: { type: Boolean, attribute: 'display-result' },
         filter: { type: String, attribute: 'filter', reflect: true },
         filtered: { type: String, attribute: 'filtered' }, // freeze filter
+        hasMore: { type: Boolean },
+        limit: { type: Number, attribute: 'limit' },
+        overrides : { type: String },
+        page: { type: Number, attribute: 'page', reflect: true },
+        resultCount: {
+          type: Number,
+        },
         search: { type: String, attribute: 'search', reflect: true },
+        sidenav: { type: Object },
+        singleApp: { type: String, attribute: 'single-app', reflect: true },
         sort: {
             type: String,
             attribute: 'sort',
             default: SORT_ORDER.authored,
             reflect: true,
         },
         types: { type: String, attribute: 'types', reflect: true },
-        limit: { type: Number, attribute: 'limit' },
-        page: { type: Number, attribute: 'page', reflect: true },
-        singleApp: { type: String, attribute: 'single-app', reflect: true },
-        hasMore: { type: Boolean },
-        displayResult: { type: Boolean, attribute: 'display-result' },
-        resultCount: {
-            type: Number,
-        },
-        sidenav: { type: Object },
     };
 
+    #overrideMap = {};
```

Comment:
This is clean and ok to me. 
However, with personalization in mind as well, sometimes in the future, I was hoping that we could replace fragments transparently without the knowledge of `aem-fragment` element.
e.g: once we know `fragment#abc` must be replaced by `fragment#xyz` we could somehow handle this in the shared cache of `aem-fragment` and the internal fetch logic.

Reply:
@yesil i agree it would be ideal, however, not really possible. For now we managed to make `aem-fragment` agnostic to what it requests and receives, and with the collection as it is today, we need that complicated payload massage at merch-card-collection level. 
May be the best would eventually move the override string (a:b,c:d,...) as a hashed value to the payload, but this will request some time (akamai configuration change). I'll track that change as a separate due diligence ticket.